demos/sslecho/main.c

```diff
@@ -23,17 +23,17 @@
 
 static const int server_port = 4433;
 
-typedef unsigned char   bool;
+typedef unsigned char   flag;
 #define true            1
```

Comment:
Do we also need to handle the definitions of true and false here?  If we start allowing the inclusion of stdbool.h, that header file defines those values:
```
#if defined __STDC_VERSION__ && __STDC_VERSION__ > 201710L
/* bool, true and false are keywords.  */
#else
#define bool    _Bool
#define true    1
#define false   0
#endif
```

Reply:
This define is not causing any issue. We do not include stdbool and preprocessor defines should not conflict with keywords.